src/index.js

```diff
@@ -23,9 +25,11 @@ export function HoloFuelApp () {
 
   return <ApolloProvider client={apolloClient}>
     <Router>
-      <ScreenWidthContext.Provider value={isWide}>
-        <HFRouter />
-      </ScreenWidthContext.Provider>
+      <HFScreenWidthContext.Provider value={isWide}>
+        <FlashMessageProvider>
```

Comment:
This stack of providers at the top of the tree is my main reservation with the `context` pattern for global state at the moment. A simple solution for the source code would be to put them all in a `<Providers>` component and import that from somewhere else, but that will still leave all those components in the actual tree, which will make debugging marginally more annoying.

Reply:
Yeah, I'll watch as we go and see what emerges here. I think for now this is all fine.